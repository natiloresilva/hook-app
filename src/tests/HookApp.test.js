import { shallow } from "enzyme"
import { HookApp } from "../HookApp"

describe('Pruebas en <HookApp/>', () => {

    test('Debe mostrarse correctamente el componente', () => {

        const wrapper = shallow(<HookApp />)
        expect(wrapper).toMatchSnapshot()

    })

})
