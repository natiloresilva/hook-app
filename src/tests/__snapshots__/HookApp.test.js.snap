// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <HookApp/> Debe mostrarse correctamente el componente 1`] = `
<div>
  <h1>
    Hola Mundo
  </h1>
</div>
`;
